src/core/store.ts

```diff
@@ -426,6 +426,10 @@ export const createStore = (
           // NOTE technically possible but restricted as it may cause bugs
           throw new Error('atom not writable')
         }
+        const aState = readAtomState(a)
```

Comment:
Can this be `getAtomState`? readAtomState tries to do more work.

Reply:
Done 👍🏼